src/app/app.routes.ts

```diff
@@ -1,6 +1,6 @@
 import { Routes } from '@angular/router';
 import { EventsComponent } from './components/events/events.component';
-import { CreateEventComponent } from './components/events/create-event/create-event.component';
+import { CreateEventComponent } from './components/event/create-event/create-event.component';
```

Comment:
the main folder should be "events", as it was, not "event"

Reply:
As we talk at the meeting, we will not change it .